extra/sigmund~/sigmund~.c

```diff
@@ -26,11 +26,6 @@ typedef double t_floatarg;
 and usable in other contexts.  The one external requirement is a real
 single-precision FFT, invoked as in the Mayer one: */
 
-#ifdef _MSC_VER /* this is only needed with Microsoft's compiler */
-__declspec(dllimport) extern
-#endif
-void mayer_realfft(int npoints, t_sample *buf);
-
```

Comment:
could this code-snippet be related to (and requierd for) compiling sigmung~ on Max/MSP?

Reply:
I admit I don't know... we should ask Miller. If so a good solution would be to include this code between `_MSC_VER` and `MSP`:
```
#ifdef MSP
#ifdef _MSC_VER /* this is only needed with Microsoft's compiler */
__declspec(dllimport) extern
#endif /* _MSC_VER */
void mayer_realfft(int npoints, t_sample *buf);
#endif /* MSP */
```